yaml-files/welcome.yaml

```diff
@@ -34,13 +34,9 @@
             - name: 5. English only.
               value: "Unfortunately, our moderators can only reliably moderate in English."
 
-            - name: 6. Don't abuse the rep system.
+            - name: 6. No griefing.
               value:
-                  "The rep system lets everyone know your 'reputation' on the server. If you've helped other people, they can upvote you, and
-                  if you've done something not so helpful like griefing, killing, etc. they can downvote you. Each action has to have a reason specified.
-                  Invalid reasons are things like 'being nice' and 'being mean', whereas valid reasons are 'they killed me and took my items' or
-                  'they helped me build my base'. Admins can check these logs, so be reasonable, or else we may have to take action against you,
-                  or reset their rep to the previous value. Fabric Reputation System can be found [here](https://modrinth.com/mod/fabric-rep-system)."
+                  "Griefing of any kind not enjoyed by the person getting griefed, including destroying of creations and stealing, is not allowed."
```

Comment:
i think `not enjoyed by the person getting greifed` should be changed to `not explicitly permitted by the person getting griefed`

i also think `including destroying of creations and stealing` should be replaced with `including destruction of another person's builds or creations, as well as stealing`

Reply:
How's this?
```suggestion
                  "Griefing of any kind not explicitly permitted by the person getting griefed, including destruction of another person's builds or creations, as well as stealing, is not allowed."
```

Reply:
based